src/Security/Http/Logout/ContextInvalidationLogoutSuccessHandler.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace FOS\HttpCacheBundle\Security\Http\Logout;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Http\Logout\LogoutHandlerInterface;
+use Symfony\Component\Security\Http\Logout\SessionLogoutHandler;
+
+class ContextInvalidationSessionLogoutHandler implements LogoutHandlerInterface
```

Comment:
should be final imho.

we should add a docblock with documentation to explain why we build this as decorator of the session logout handler rather than a stand-alone handler.